x-pack/legacy/plugins/siem/public/containers/case/api.ts

```diff
@@ -130,3 +130,25 @@ export const patchComment = async (
   await throwIfNotOk(response.response);
   return convertToCamelCase<CommentResponse, Comment>(decodeCommentResponse(response.body));
 };
+
+export const deleteCases = async (caseIds: string[]): Promise<boolean> => {
+  const response = await KibanaServices.get().http.fetch<string>(`${CASES_URL}`, {
+    method: 'DELETE',
+    asResponse: true,
+    query: { ids: JSON.stringify(caseIds) },
+  });
+  await throwIfNotOk(response.response);
+  return response.body === 'true' ? true : false;
+};
+
+export const deleteComment = async (caseId: string, commentId: string): Promise<boolean> => {
```

Comment:
What do you think about passing a signal (AbortController) so we can pass it to `fetch`

Reply:
resolving this one because i removed `deleteComment`